### PR TITLE
Issue #41 - Documentation reviewed, made concise and beginners friendly, added a JupyterHub image

### PR DESCRIPTION
**Pull request by alexanderchosen**
Hello mentors @GeorgianaElena @sgibson91 @minrk, I properly reviewed the config-user-env.md under issue #41 by simplifying some terms used to enable all kinds of users, especially beginners easily understand. I also added a JupyterHub image at the top to give the documentation a better-friendly look.
I look forward to your feedback. Thanks

### CODE REVIEW

File: docs/source/reference/config-user-env.md [hunk 1]
```diff
@@ -1,21 +1,22 @@
+![jupyterhub](https://jupyter.org/assets/homepage/hublogo.svg)
+
```
Comment (sgibson91) [inline docs/source/reference/config-user-env.md:2]: I don't think this is needed since our documentation theme will always put our logo in the top menu bar anyway

Comment (sgibson91) [inline docs/source/reference/config-user-env.md:2]: I don't think this is necessary, since our documentation theme will add our logo to the top bar on all pages

Comment (alexanderchosen) [reply docs/source/reference/config-user-env.md:2]: Thanks for the feedback. I will remove the logo at the top bar.
Are there any other corrections to note and make changes?

**Comment by alexanderchosen** [comment]
@sgibson91 please I need your help to clarify the conflicts. I removed the image as instructed and I made a few changes to the text structure format. I still see the conflicting files flag and checks error. Please what should I do next to resolve them?
Thanks a lot.

**Comment by alexanderchosen** [comment]
@sgibson91 thanks for your great help. I was able to resolve the conflicts and also pass all checks.

File: docs/source/reference/config-user-env.md [hunk 2]
```diff
@@ -141,31 +122,25 @@ How you configure user environments for each category can differ a bit
...
 
 JupyterHub can, however, have multiple servers per user.
-This is most useful in deployments where users can configure the environment
-in which their server will start (e.g. resource requests on an HPC cluster),
-so that a given user can have multiple configurations running at the same time,
-without having to stop and restart their one server.
+This is mostly useful in deployments where users can configure the environment in which their server will start (e.g. resource requests on an HPC cluster), so that a given user can have multiple configurations running at the same time, without having to stop and restart their own server.
 
-To allow named servers:
+To allow named servers, use this:
```
Comment (sgibson91) [inline docs/source/reference/config-user-env.md:143]: I'd rephrase to something like "include this code snippet in your config file". "use this" isn't very explanatory.

File: docs/source/reference/config-user-env.md [hunk 3]
```diff
@@ -24,51 +21,41 @@ This section will focus on user environments, including:
...
-environments for all JupyterHub users. For example, you wish for each student in
-a class to have the same user environment configuration.
-
-Jupyter and IPython support **"system-wide"** locations for configuration, which
-is the logical place to put global configuration that you want to affect all
-users. It's generally more efficient to configure user environments "system-wide",
-and it's a good idea to avoid creating files in users' home directories.
+As a JupyterHub administrator, you will typically want to install and configure environments for all JupyterHub users. For example, let's say you wish for each student in a class to have the same user environment configuration.
 
+Jupyter and IPython support **"system-wide"** locations for configuration, which is the logical place to put global configuration that you want to affect all users. It's generally more efficient to configure user environments "system-wide",and it's a good practice to avoid creating files in the users' home directories.
```
Comment (sgibson91) [inline docs/source/reference/config-user-env.md:50]:
```suggestion
Jupyter and IPython support **"system-wide"** locations for configuration, which is the logical place to put global configuration that you want to affect all users. It's generally more efficient to configure user environments "system-wide", and it's a good practice to avoid creating files in the users' home directories.
```

File: docs/source/reference/config-user-env.md [hunk 4]
```diff
@@ -183,13 +158,13 @@ as well as the admin page:
 Named servers can be accessed, created, started, stopped, and deleted
 from these pages. Activity tracking is now per server as well.
 
-The number of named servers per user can be limited by setting a constant value:
+To limit the number of **named server** per user by setting a constant value, use this:
```
Comment (sgibson91) [inline docs/source/reference/config-user-env.md:161]: Again, "use this" isn't very clear

File: docs/source/reference/config-user-env.md [hunk 5]
```diff
@@ -183,13 +158,13 @@ as well as the admin page:
...
 
 ```python
 c.JupyterHub.named_server_limit_per_user = 5
 ```
 
-or a callable/awaitable based on the handler object:
+Alternatively, to use a callable/awaitable based on the handler object, use this:
```
Comment (sgibson91) [inline docs/source/reference/config-user-env.md:167]: Another "use this"

**Comment by alexanderchosen** [comment]
Thanks alot @sgibson91 for the corrections. I will make necessary changes and ask for your review.

**Comment by alexanderchosen** [comment]
@sgibson91 thanks for the corrections. I have made the necessary changes and reviewed the documentation again.

**Review by sgibson91 — APPROVED** [review]
Thank you @alexanderchosen - Don't forget to move your handle into the "Done" column of the relevant spreadsheet now!

**Comment by alexanderchosen** [comment]
Thanks for your guidance and patience @sgibson91. I have moved my handle to the done column in the spreadsheet.
